src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -250,14 +251,30 @@ def validateOutputDatasets(outDsets, dbsUrl):
     _validateDatatier(datatier, dbsUrl)
 
 
-def _validateDatatier(datatier, dbsUrl):
+@retry(tries=5, delay=1)
+def _listDatatiers(dbsUrl):
+    """
+    Query DBS for list of data tiers.
+    If exception is raised, retry up to 5 times with a 1 second delay.
+    """
+    dbsTiers = DBSReader.listDatatiers(dbsUrl)
+    return dbsTiers
+
+def _validateDatatier(datatier, dbsUrl, expiration=3600):
     """
     _validateDatatier_
 
     Provided a list of datatiers extracted from the outputDatasets, checks
     whether they all exist in DBS.
     """
-    dbsTiers = DBSReader.listDatatiers(dbsUrl)
+    mc = MemoryCacheStruct(1, _listDatatiers, kwargs={'dbsUrl': dbsUrl})
```

Comment:
Didn't you want to set `expiration` to the `expire` parameter?

Reply:
Absolutely yes, I must have hardcoded it to 1 when debugging and then forgot to change it back. Will fix